main.py

```diff
@@ -244,7 +246,9 @@ def _set_secret(config_filename):
     if 'secret' in Configuration and Configuration['secret'] is not None:
         secret = Configuration['secret']
     else:  # No secret found, generate one and persist it
-        secret = hashlib.sha512().hexdigest()
+        secret_length = 128
+        secret = ''.join(random.SystemRandom().choice(string.ascii_lowercase + string.digits)
+                         for _ in range(secret_length))
```

Comment:
This passes linting just fine and is the default in IntelliJ but just for you TJ I can add a few more lines to reduce the line length.